docs/Salesforce-streamingsource.md

```diff
@@ -56,14 +60,14 @@ This field not required if you are using an existing push topic.
 
 
 **Notify For Fields:** Push topic property, which specifies how the record is evaluated against the
-PushTopic query. The NotifyForFields values are:<br>
+PushTopic query. The NotifyForFields values are:  
```

Comment:
Change NotifyForFields to Notify For Fields

Reply:
done